splink/comparison_template_library.py

```diff
@@ -35,17 +36,15 @@ def __init__(
         self,
         col_name: Union[str, ColumnExpression],
         *,
-        date_format: str = None,
+        datetime_thresholds: Union[int, float, List[Union[int, float]]],
+        datetime_metrics: Union[DateMetricType, List[DateMetricType]],
+        datetime_format: str = None,
         invalid_dates_as_null: bool = False,
         include_exact_match_level: bool = True,
+        input_is_string: bool = False,
         separate_1st_january: bool = False,
-        fuzzy_metric: str = "damerau_levenshtein",
-        fuzzy_thresholds: Union[float, List[float]] = [1],
-        datetime_thresholds: Union[int, List[int]] = [1, 1, 10],
-        datetime_metrics: Union[str, List[str]] = ["month", "year", "year"],
+        use_damerau_levenshtein: bool = True,
```

Comment:
damerau_levenshtein is probably theoretically more sound, but unavailable in postgres.  I've added this argument to make sure users can use the DateComparison even if they're using postgres.  

I could have been 'cleverer' and made it automagically use levenshtein in the case of postgres, but it adds complexity and it's not obvious it's a good idea to change functionality magically like that between backends